syntax = "proto3";

import "data_object_config.proto";

message ParamsConfig {
    map<string, DataObjectConfig> params = 1;
}
